### PR TITLE
Add shield for Western Ring Route in New Zealand

### PR DESCRIPTION
**Pull request by k-yle**
This PR adds a shield for `network=NZ:WRR`, which is a one-of-a-kind route ([example 1](https://www.nzta.govt.nz/media-releases/new-logo-to-guide-motorists-on-western-ring-route/), [example 2](https://goo.gl/maps/4Thd4qVTCpHrtQmo6))

Related to #451

### CODE REVIEW

**Comment by ZeLonewolf** [comment]
Thanks @k-yle for opening a PR.

Looking at this image:
image

I have questions.

Is the WRR route relation separate from the route 18 relation?  I would think that we would want a custom graphic for the route, much like we do for special turnpikes in the United States.  This PR declares a pill shield, which draws a circle or pill around the value in `ref`, and I'm not sure that's really the behavior we're looking for here?

**Comment by k-yle** [comment]
yes, it's separate from SH 18, ([this is the OSM relation](https://osm.org/relation/13730708)). I saw that `US:IA` uses the same style, and [renders similarly](https://zelonewolf.github.io/openstreetmap-americana/#13.81/42.22584/-91.86094) to how the WR shields are signposted. 

I wasn't sure if a custom graphic was appropriate in this case (?), since it's a pretty much just a circle with text inside.

**Comment by ZeLonewolf** [comment]
Oh, I see.  Black on white WR in a pill does seem like a workable approximation in that case.

File: src/js/shield_defs.js [hunk 1]
```diff
@@ -3180,6 +3180,7 @@ export function loadShields(shieldImages) {
     },
   };
   shields["NZ:UR"] = homeDownShield;
+  shields["NZ:WRR"] = pillShield(Color.shields.white, Color.shields.black);
```
Comment (claysmalley) [inline src/js/shield_defs.js:3183]:
```suggestion
  shields["NZ:WRR"] = ovalShield(Color.shields.white, Color.shields.black, Color.shields.black, 20);
```

**Comment by ZeLonewolf** [comment]
Code looks essentially correct, needs a tile update to test.

**Review by claysmalley — CHANGES_REQUESTED** [review]
I think the plain text approach is reasonable, though it at least seems important that the shield is a circle rather than a pill or oval. If we use `ovalShield()` and specify a width of 20 it should make the shield circular.

**Review by claysmalley — APPROVED** [review]
Looks good!

Screenshot from 2022-08-16 17-58-41

**Comment by ZeLonewolf** [comment]
Thanks @k-yle for your contribution!

**Pull request merged**